source/guides/integration.rst

```diff
@@ -11,6 +11,7 @@ See one of the sample integrations below or visit the `Mattermost Integrations D
    
    /integrations/jira*
    /integrations/zoom*
+   /integrations/net-promoter*
```

Comment:
It's technically an integration since it's a plugin, but I don't know if admins would know to look there for it

Reply:
Yea technically that makes sense, but from the admin's point of view, they don't necessarily view it as an "integration".

E.g. if we built DND Scheduling as a plugin, it likely wouldn't be documented as an integration, it would be documented as an end user feature.

Reply:
Perhaps under the Admin guide > Mattermost Integrations section? Kind of 0/5 on where is the best pace for this. Seems the Telemetry docs are not even in the TOC

Reply:
@wiersgallak Wondering if you'd have an opinion from admin's perspective.

I'd lean towards adding it to https://docs.mattermost.com/guides/administrator.html#administration

Reply:
For now I added it to Admin guide > Mattermost Integrations in addition to keeping it in the Integrations guide since I think some admins might look for it there given it's a plugin. Open to other suggestions